proposals/p1885.md

```diff
@@ -81,6 +85,12 @@ The original `for` proposal does not define a way for user-defined types to
 opt-in to support range-based for loops. An approach is proposed in this
 document.
 
+This can be achieved using either a cursor or an iterator. A cursor benefits
```

Comment:
It's probably worth spelling out the difference between "cursor" and "iterator"; I'm not sure everyone is familiar with that terminology.